wallet/chain/p/builder/context.go

```diff
@@ -69,6 +71,25 @@ func NewContextFromClients(
 	}, nil
 }
 
+func NewContextFromConfig(
+	ctx *snow.Context,
+	cfg *config.Config,
+	timestamp time.Time,
+) *Context {
+	return &Context{
+		NetworkID:                     ctx.NetworkID,
+		AVAXAssetID:                   ctx.AVAXAssetID,
+		BaseTxFee:                     cfg.TxFee,
+		CreateSubnetTxFee:             cfg.GetCreateSubnetTxFee(timestamp),
+		TransformSubnetTxFee:          cfg.TransformSubnetTxFee,
+		CreateBlockchainTxFee:         cfg.GetCreateBlockchainTxFee(timestamp),
+		AddPrimaryNetworkValidatorFee: cfg.AddPrimaryNetworkValidatorFee,
+		AddPrimaryNetworkDelegatorFee: cfg.AddPrimaryNetworkDelegatorFee,
+		AddSubnetValidatorFee:         cfg.AddSubnetValidatorFee,
+		AddSubnetDelegatorFee:         cfg.AddSubnetDelegatorFee,
+	}
+}
```

Comment:
i would keep this in the platformvm txbuilder package. It's not really needed outside of it (and a couple of other UTs).